docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -39,7 +39,7 @@ By contrast, an **active-passive** setup designates one region as the main or ac
 
 <DualRegion />
 
-The illustrated architecture consists of two regions. Each region houses a Kubernetes cluster in which Camunda 8 is deployed. Those two Kubernetes clusters are capable of communicating with each other.
+The illustrated architecture consists of two regions. For illustrative purposes, we're showing Kubernetes-based installation. Each region houses a Kubernetes cluster with Camunda 8 deployment. Those two Camunda 8 setups are capable to communicate with each other.
```

Comment:
Maybe `depicted` instead of having `illustrated` followed by `illustrative`

```
The depicted architecture consists of two regions. For illustrative ...
```

Reply:
Good point, fixing it.